libs/rapidsandrivers-extras/src/main/kotlin/rapidsandrivers/Kontekst.kt

```diff
@@ -0,0 +1,9 @@
+package rapidsandrivers
+
+enum class Kontekst {
+    BREV,
```

Comment:
Finnes det ikke rivere for andre behandlingsting også som ikke direkte omhandler brev? Mulig de ikke implementerer feilhaandtering på denne måten?

Reply:
Stemmer! Dei andre byggjar på `ListenerMedLogging`-superklassa, som ikkje har feilhandteringsbiten. Den laga vi eigentleg mest for regulering og migrering, og er vel også fin for andre kontekstar kor éin feil ikkje skal stoppe resten av flyten for dei som kjem etter.